README.md

```diff
@@ -47,4 +47,53 @@ mode of the feed export:
 - `overwrite=False` and `blob_type="AppendBlob"` append to the blob if it 
   exists and it is an `AppendBlob`, and create it otherwise. 
 - `overwrite=True` overwrites the blob, even if it exists. The `blob_type` must
-  match that of the target blob.
\ No newline at end of file
+  match that of the target blob.
+
+## Store any data in Azure Blob Storage
+It is possible to store any kind of data directly to Azure Blob Storage just
+by using the `AZURE_ACCOUNT_URL` and `AZURE_ACCOUNT_KEY` settings, also the `FILE_STORE`
+or `IMAGE_STORE` settings with the URI to the Azure Blob Storage account or the local
+container provided by [Azurite](https://github.com/Azure/Azurite).
+
+### Usage
+Suppose that you want to extract all images from a website and save it into a folder in
+your Azure Blob Storage, but first you need to test it locally using Azurite.
+
+1. In the project settings do the initial setup.
+
+```python
+FILES_STORE = "http://127.0.0.1:10000/<account>/<container>"
+AZURE_ACCOUNT_URL = "http://127.0.0.1:10000/<account>"
+AZURE_ACCOUNT_KEY = "<account_key>"
+```
+
+2. Create your own Pipeline
+
+```python
+from scrapy.pipelines.files import FilesPipeline as FSPipeline
+from scrapy_azure_exporter.azure_store import AzureFilesStore
+
+
+class FilesPipeline(FSPipeline):
+    @classmethod
+    def from_settings(cls, settings):
+        azure_store = AzureFilesStore
+        azure_store.AZURE_CONNECTION_STRING = settings.get("AZURE_CONNECTION_STRING")
+        azure_store.AZURE_ACCOUNT_URL_WITH_SAS_TOKEN = settings.get("AZURE_ACCOUNT_URL_WITH_SAS_TOKEN")
+        azure_store.AZURE_ACCOUNT_URL = settings.get("AZURE_ACCOUNT_URL")
+        azure_store.AZURE_ACCOUNT_KEY = settings.get("AZURE_ACCOUNT_KEY")
+
+        # NOTE: The "http" schema is necessary to compatibility with Azurite
+        cls.STORE_SCHEMES.update({"azure": azure_store, "http": azure_store})
+        return cls(settings.get("FILES_STORE"), settings=settings)
```

Comment:
I don’t like much of this approach, but [I have now realized it is how Scrapy works](https://github.com/scrapy/scrapy/issues/5991), so I guess there is not much to do about it.

I am now starting to see why you originally decided to provide a FilesPipeline subclass, and I am starting to think it makes sense, although I think we can have both.

We could have a `azure_storage_from_settings` function that gets `(settings)` and returns a class that can be assigned to `STORE_SCHEMES`. That way, this code could be simplified as:

```
class MyFilePipeline(FilesPipeline):
    @classmethod
    def from_settings(cls, settings):
        pipeline = super().from_settings(settings)
        pipeline.STORE_SCHEMES["azure"] = azure_storage_from_settings(settings)
        return pipeline
```

And you could provide `AzureFilesPipeline` and `AzureImagesPipeline` implementations that use this approach, recommend using them when possible to keep things simple, but still cover this alternative approach for people that need it.

Reply:
For clarification, that `azure_storage_from_settings` function could create a `AzureFuleStore` on the spot, so that we do not need to modify the base class. Something like:

```python
def azure_storage_from_settings(settings):
    class ConfiguredAzureFileStore(AzureFileStore):
        foo = settings["bar"]

    return ConfiguredAzureFileStore
```

I think this approach is the cleanest we can get until we address https://github.com/scrapy/scrapy/issues/5991 upstream in the future.